dev/com.ibm.ws.security.kerberos.java8/src/com/ibm/ws/security/kerberos/Krb5HelperJdk11.java

```diff
@@ -57,8 +56,11 @@
     /**
      * We don't do anything with the process, but having it set allows us to only be activated by DS if criteria we set
      * about the Java version are met.
+     * 1) IBM JDK 1.8 has value added and the vendor name can be IBM or not IBM
+     * 2) IBM JDK 11 use SUN Krb5LoginModule and the vendor name can be IBM or not IBM
      */
-    @Reference(policy = ReferencePolicy.STATIC, target = "(&(java.specification.version>=1.8)(!(java.vendor=ibm corporation)))")
+    @Reference(policy = ReferencePolicy.STATIC,
+               target = "(|(&(java.specification.version>=1.8)(java.specification.version<=9.0)(!(java.vendor=ibm corporation)) (java.specification.version>=9.0))))")
```

Comment:
Can it be reduced to:

(|(java.specification.version>1.8)(!(java.vendor=ibm corporation)))

Reply:
Yes,  Liberty support JDK 1.8 and higher.

Reply:
yes